### PR TITLE
Update Installation Guides

### PR DESCRIPTION
**Pull request by U8NWXD**

### CODE REVIEW

**Comment by DubeySandeep** [comment]
> The changes to the Windows instructions are minimal since I don't have a Windows machine to test them on

Do we know any developer who is currently working with windows? If yes, can you talk to them?

**Comment by DubeySandeep** [comment]
@U8NWXD Is this PR ready for review? We had discussed for some changes in this PR, right?

**Comment by U8NWXD** [comment]
@DubeySandeep I've removed the example `start.py` outputs as we discussed. PTAL

**Comment by U8NWXD** [comment]
@DubeySandeep I've updated my changes for python 3 and addressed your comments. PTAL

File: Installing-Oppia.md [hunk 1]
```diff
@@ -1,10 +1,59 @@
...
-   * [Mac OS](https://github.com/oppia/oppia/wiki/Installing-Oppia-%28Mac-OS%29)
-   * [Windows](https://github.com/oppia/oppia/wiki/Installing-Oppia-%28Windows%29)
-4. If you run into any problems during installation, please read [these notes](https://github.com/oppia/oppia/wiki/Issues-with-installation%3F) and the [troubleshooting](https://github.com/oppia/oppia/wiki/Troubleshooting) page.
+Before working through these instructions, you should read through our [[instructions for code contributors|Contributing-code-to-Oppia]]. Once you've done that, you're ready to start installing Oppia!
+
+If you are installing on Windows, skip to the [[Windows installation guide|Installing-Oppia-(Windows)]]. The following instructions do not apply to Windows installations.
+
+1. Create a new, empty folder. We'll call it `opensource/`.
+
+1. Navigate to the folder (`cd opensource/`). Next, we'll [fork and clone](https://help.github.com/articles/fork-a-repo/) the Oppia repository.
```
[the remark this answers is not in the document]
Comment (U8NWXD) [reply Installing-Oppia.md]: done

**Review by DubeySandeep — COMMENTED** [review]
LGTM!

@vojtechjelinek Can you take a final pass? (As you all have recently made changes in these wiki pages, so your review will be very helpful!)

File: Installing-Oppia-(Linux;-Python-3).md [hunk 2]
```diff
@@ -1,96 +1,274 @@
...
+
+   We have established a clean setup now. We can make any changes we like and push it to this forked repository, and then make a pull request for getting the changes merged into the original repository. Here's a nice picture explaining the process ([image source](https://github.com/Rafase282/My-FreeCodeCamp-Code/wiki/Lesson-Save-your-Code-Revisions-Forever-with-Git)).
+
+   ![Diagram of the fork-and-clone workflow](images/install/forkCloneWorkflow.png)
+
+   For making any changes to original repository, we first sync our cloned repository with original repository. We merge develop with `upstream/develop` to do this. Now we make a new branch, do the changes on the branch, push the branch to forked repository, and make a PR from Github interface. We use a different branch to make changes so that we can work on multiple issues while still having a clean version in develop branch.
+
+## Setup a virtual environment
+
+For your vitual environment, we recommend you use [pyenv](https://github.com/pyenv/pyenv). Here are some instructions for doing so, but you can use another virtual environment tool if you wish:
```
Comment (vojtechjelinek) [inline Installing-Oppia-(Linux;-Python-3).md:88]:
```suggestion
For your virtual environment, we recommend you use [pyenv](https://github.com/pyenv/pyenv). Here are some instructions for doing so, but you can use another virtual environment tool if you wish:
```

**Review by vojtechjelinek — APPROVED** [review]
Thanks! One nit.

File: Installing-Oppia-(Linux;-Python-3).md (hunk 2 above)
Comment (U8NWXD) [reply Installing-Oppia-(Linux;-Python-3).md]: done

**Pull request merged**